v2/pkg/vm/qemu.go

```diff
@@ -139,6 +141,12 @@ func (c *qemu) qemuParams(r *Runtime) []string {
 	if c.memory != "" {
 		params = append(params, "-m", c.memory)
 	}
+	if c.numa.nodes != 0 {
+		cpuPerNode := c.smp.cpus / c.numa.nodes
+		for i := 0; i < c.numa.nodes; i++ {
+			params = append(params, "-numa", fmt.Sprintf("node,cpus=%d-%d", cpuPerNode*i, cpuPerNode*(i+1)-1))
```

Comment:
I'm not sure whether this is valid for the latest QEMU.
It would be OK to care only after a user has complained about this.
https://www.qemu.org/docs/master/about/removed-features.html#numa-node-without-memory-specified-removed-in-5-2

Reply:
Hmm... it may seem a problem. Ubuntu 20.04 uses QEMU 4.2 but Ubuntu 22.04 uses QEMU 6.2.